### PR TITLE
Fix: Manage verified users for multi-sig

### PR DESCRIPTION
**Pull request by mmioana**
## Description

Create and finalize multi-sig add and remove verified members motions.

- [X] The add verified members multi-sig motion can be created
- [X] The remove verified members multi-sig motion can be created
- [X] The add verified members motion can be finalized
- [X] The remove verified members motion can be finalized
- [X] Only users with [adequate permissions](https://www.notion.so/colony/Multi-sig-Permissions-6aaf0b95309f4754a243d54e8d9c4189?pvs=4#fec3699a5fd241a2925bd2809494b157) can create them

[CDapp PR](https://github.com/JoinColony/colonyCDapp/pull/2650)

### CODE REVIEW

File: src/handlers/multiSig/multiSigCreated/multiSigCreated.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
   handleMintTokensMultiSig,
   handleUnlockTokenMultiSig,
 } from './handlers';
+import { handleMetadataDeltaMultiSig } from './handlers/metadataDelta';
```
Comment (bassgeta) [inline src/handlers/multiSig/multiSigCreated/multiSigCreated.ts:16]: maybe we should import it from the index file?

**Review by bassgeta — CHANGES_REQUESTED** [review]
Tested as part of [CDapp](https://github.com/JoinColony/colonyCDapp/pull/2650), works fine, dropped a few comments there and a minor import related one here.
Nice and straightforward changes!

**Review by iamsamgibbs — APPROVED** [review]
Looks great to me!

**Review by bassgeta — APPROVED** [review]
Tested as part of [CDapp PR](https://github.com/JoinColony/colonyCDapp/pull/2650/files) and it works as expected, nice one!

**Pull request merged**